lib/gossip/dissemination.js

```diff
@@ -130,6 +152,7 @@ Dissemination.prototype.onRingChanged = function onRingChanged() {
 
 Dissemination.prototype.recordChange = function recordChange(change) {
     this.changes[change.address] = change;
+    this.changes[change.address].piggybackCount = 0;
```

Comment:
This modifies the caller-provided change... which is not great from an API perspective.